### PR TITLE
Added Gitbutler

### PR DESCRIPTION
**Pull request by Ajay-056**
# Pull Request

## Title
Added Gitbutler

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [x] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
Gitbutler is a git client for simultaneous branches on top of your existing workflow.

## Testing
[Detail the testing you have performed to ensure that these changes function as intended. Include information about any added tests.]

## Impact
[Discuss the impact of your changes on the project. This might include effects on performance, new dependencies, or changes in behaviour.]

## Issue related to PR
[What issue/discussion is related to this PR (if any)]
- Resolves #

## Additional Information
[Any additional information that reviewers should be aware of.]

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: config/applications.json [hunk 1]
```diff
@@ -727,6 +727,14 @@
 		"link": "https://git-scm.com/",
 		"winget": "Git.Git"
 	},
+	"gitbutler": {
+		"category": "Development",
+		"choco": "na",
+		"content": "Git GUI",
```
Comment (MyDrift-user) [inline config/applications.json:733]: The "content" is the name that the application is displayed as, may I ask why it is named "Git GUI" and not  "GitButler"?

**Review by MyDrift-user — APPROVED** [review]
application looks cool, licence is `FSL-1.1-MIT` and install works as expected.

File: config/applications.json (hunk 1 above)
Comment (Ajay-056) [reply config/applications.json]: Yeah it should be, Thanks for spotting it.

**Review by og-mrk — APPROVED** [review]
Looks good. Thanks for the cool addition @ajay-056 👍

**Review by Real-MullaC — APPROVED** [review]

**Pull request merged**